test.txt

```diff
@@ -0,0 +1,14 @@
+class Doughnut {
```

Comment:
i reduced your test case because the thing that was failing was the `super.cook()` call